thorlcr/activities/keyedjoin/thkeyedjoinslave.cpp

```diff
@@ -907,6 +911,7 @@ class CKeyedJoinSlave : public CSlaveActivity, implements IJoinProcessor, implem
         }
         void block()
         {
+            BlockedActivityTimer t(activity.getTotalCyclesRef(), activity.queryTimeActivities());
```

Comment:
same q. as per prev. comments.
Need to think carefully whether overlapping concurrent threads that are pending on something are blocking the acts start/nextrow from making progress.